crates/turborepo-lib/src/shim.rs

```diff
@@ -221,8 +221,12 @@ impl RepoState {
     }
 
     fn spawn_local_turbo(&self, local_turbo_path: &Path, mut shim_args: ShimArgs) -> Result<i32> {
-        let cwd = self.root.canonicalize()?;
+        println!(
```

Comment:
Should this be debug level?

Reply:
Do we show debug level by default? I want this to be shown regardless because I got really confused just now when I ran `turbo7 --help` and got the old help screen.

Reply:
We don't, but yea that makes sense. We should be explicit here.